pycommon/reporter_config/actions/Email.py

```diff
@@ -11,15 +15,17 @@ class EmailAction(Action):
     Mail is send by localhost SMTP server. In the future this
     should be configurable in the config.
 
-    Parameter "subject" may contain variables that are substituted by 
+    Parameter "subject" may contain variables that are substituted by
```

Comment:
This part of doc is not up-to-date because subject is now rendered by jinja as well as mail body.

I suggest to "shorten" this text to minimal information and a comprehensive text will be prepared at https://nemea.liberouter.org/reporting/

Reply:
Díky za změny, ale ještě tam jsou ty parametry `forceSSL` a `startTLS`.

Reply:
Pardon zapomněl jsem je tam přidat. V novém commitu jsou už parametry forceSSL a startTLS v smtp_connection